lib/ueberauth.ex

```diff
@@ -69,7 +69,7 @@ defmodule Ueberauth do
   In phoenix setup a pipeline:
 
       pipeline :ueberauth do
-        Ueberauth.plug "/auth"
+        plug Ueberauth, base_path: "/auth"
```

Comment:
the plug needs to continue downstream, but you do need to define a request route so the pipeline gets fired atm